chicagobub/scripts/bubble_chart.js

```diff
@@ -107,12 +133,16 @@ function bubbleChart() {
 
   function ticked() {
     bubbles
-      .attr('cx', function (d) { return d.x; })
-      .attr('cy', function (d) { return d.y; });
+      .attr('cx', function (d) {
+        return d.x;
+      })
+      .attr('cy', function (d) {
+        return d.y;
+      });
   }
 
   function nodeYearPos(d) {
-      return yearCenters[d.year].x;
+    return yearCenters[d[yearToSplitOn]].x;
```

Comment:
now instead of just getting `d.year` - which in the original code was always `INC1970`, we instead use the `yearToSplitOn` global - which is modified every time a user clicks a different year button.